config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,15 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// OpenStackPlatformStatus holds the current status of the OpenStack infrastructure provider.
+type OpenStackPlatformStatus struct {
+	// APIVIP is the VRRP address for the internal OpenShift API access.
+	APIVIP string `json:"apiVip,omitempty`
+
+	// DNSVIP is the VRRP address for the internal cluster DNS.
+	DNSVIP string `json:"dnsVip,omitempty`
```

Comment:
@tomassedovic can you point to what needs to consume these fields?

Reply:
I didn't have time to put up proper PRs for this yet (plan to do that tomorrow), but this is how it's intended to be used (from my in-development branches I tested today, the IPs are hardcoded, etc. for now):

https://github.com/tomassedovic/installer/commit/74bff07833619f17ce595d92b65468ae0f07c775

https://github.com/openshift/machine-config-operator/commit/0490406cfeaefb75bd00e326e8d4f323cc8d815a#diff-401a0f8b1d8edd607fbaf7c85beba53a

This is more or less analogous to what @rgolangh is doing with oVirt: https://github.com/openshift/api/pull/369 and I believe the baremetal IPI folks will want to do in their case too. We need these fields to pass VIPs from the installer to static pods running on the nodes.

Since we've got full control over the field names, we can change those if they don't fit the project's style.